.github/workflows/update-tpp-schema.yml

```diff
@@ -30,14 +30,20 @@ jobs:
         id: create_pr
         uses: peter-evans/create-pull-request@67ccf781d68cd99b580ae25a5c18a1cc84ffff1f # v7.0.6
         with:
-          add-paths: tests/lib/tpp_schema.*
+          add-paths: tests/lib/*
```

Comment:
on the face of it, this looks like it will catch too many files?

Reply:
That's what I thought! (https://bennettoxford.slack.com/archives/C069YDR4NCA/p1737395525574229) but because this only gets called after `update_tpp_schema.py`, the only things that will have changed are related to that, and we want to commit everything that's changed.

Reply:
aha yes!